session_3/blogspot/.browserslistrc

```diff
@@ -0,0 +1 @@
+defaults
```

Comment:
Please don't push session 3 and 4 again

Reply:
How do I do that? I tried fetching upstream, but that didn't change the locally modified files. Should I manually delete the files?